tests/cpp_unit_tests/test_deserializer.cpp

```diff
@@ -435,10 +435,12 @@ TEST_CASE("Deserializer with error") {
             R"({"version": "1.0", "type": "input", "is_batch": false, "attributes": {}, "data": {"node": [[5]]}})";
         check_error(unequal_attributes, "Position of error: data/node/0");
         constexpr std::string_view wrong_type_list =
-            R"({"version": "1.0", "type": "input", "is_batch": false, "attributes": {"node": ["id"]}, "data": {"node": [[true]]}})";
+            R"({"version": "1.0", "type": "input", "is_batch": false, "attributes": {"node": ["id"]}, "data": {"node":
+[[true]]}})";
```

Comment:
why was this change necessary? it's a raw string, so it may not produce the exact same result (newline characters matter). It's good to test this edge case, but I feel like line length is the wrong reason to do that.

Reply:
The new clang-format automatically formatted this. I guess that's a upgrade of clang-format?

Reply:
interesting